application/controllers/User.php

```diff
@@ -69,50 +69,34 @@ public function view_program(){
 	// o pelatis prosthetei to varos tou
 	public function add_weight(){
```

Comment:
Previously, the add_weight controller was always validating the forms. 

I've changed it in order to validate the form, only when you press the button

Also, I've included an error message